dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -179,6 +179,19 @@ const getImage = ({
 	return undefined;
 };
 
+const decideSublinkPosition = (
+	supportingContent?: DCRSupportingContent[],
+	imagePosition?: ImagePositionType,
+): 'inner' | 'outer' | 'none' => {
+	if (!supportingContent || supportingContent.length === 0) {
+		return 'none';
+	}
+	if (imagePosition === 'top') {
+		return 'outer';
+	}
+	return supportingContent.length > 2 ? 'outer' : 'inner';
```

Comment:
I think I'd personally find it slightly easier to follow the logic here if the second `if` clause was `(imagePosition === 'top' || supportingContent.length > 2)`. So that the conditions which yield `'outer'` are grouped, and it's clear that the fall-through option is `'inner'`. This is just a nitpick though; feel free to disregard!